packages/xdl/src/Project.ts

```diff
@@ -1560,7 +1578,7 @@ export async function buildAsync(
 ): Promise<BuildStatusResult | BuildCreatedResult> {
   /**
     This function corresponds to an apiv1 call and is deprecated.
-    Use 
+    Use
```

Comment:
Oh my bad, I think this is my vscode removing trailing whitespaces. I can revert this if you want!

Reply:
nbd